cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -322,13 +333,53 @@ func TestEvaluateExecute(t *testing.T) {
 				ExpectedPanic: "ERROR: model openrouter/auto does not exist. Valid models are: symflower/symbolic-execution",
 			})
 		})
+		t.Run("Ollama", func(t *testing.T) {
+			validate(t, &testCase{
+				Name: "Pulled Model",
+
+				Before: func(t *testing.T, logger *log.Logger, resultPath string) {
+					t.Setenv("OLLAMA_MODELS", filepath.Join(resultPath, "TESTIGNORE-ollama-models"))
+					shutdown, err := tools.StartOllamaServer(logger, tools.OllamaPath)
+					require.NoError(t, err)
+
+					_, err = util.CommandWithResult(context.Background(), logger, &util.Command{
+						Command: []string{
+							tools.OllamaPath,
+							"pull", "qwen:0.5b",
+						},
+					})
+					require.NoError(t, err)
+					require.NoError(t, shutdown())
```

Comment:
I think the best action for the service is to make it not only dependent on the binary, but first of all on the host URL. Reason is that we can then use the service right away over HTTP.